docs/package.json

```diff
@@ -15,10 +15,14 @@
 		"build": "npm run build:api && npm run build:md-magic && npm run hugo",
 		"build:api": "npm run build:api-rollup && npm run build:api-documentation",
 		"build:api-documentation": "node ./api-markdown-documenter/index.js",
-		"build:api-rollup": "rimraf ./_api-extractor-temp && node ./rollup-api-json.js ../_api-extractor-temp/doc-models/ ./_api-extractor-temp/",
+		"build:api-documentation:multi-version": "node ./api-markdown-documenter/index.js true",
+		"build:api-rollup": "node ./rollup-api-json-helper.js",
+		"build:api-rollup:multi-version": "node ./rollup-api-json-helper.js true",
+		"build:api:multi-version": "npm run build:api-rollup:multi-version && npm run build:api-documentation:multi-version",
 		"build:md-magic": "concurrently npm:build:md-magic:code npm:build:md-magic:website",
 		"build:md-magic:code": "markdown-magic --files \"**/*.md\" !docs --workingDirectory ..",
 		"build:md-magic:website": "markdown-magic",
+		"build:multi-version": "npm run build:api && npm run build:api:multi-version && npm run build:md-magic && npm run hugo",
```

Comment:
Does this need to run both `build:api` _and_ `build:api:multi-version`? Shouldn't it only need the multi-version variant?